state/unit_ops.go

```diff
@@ -230,5 +230,25 @@ func (op *unitSetStateOperation) getUniterStateQuotaChecker() quota.Checker {
 	)
 }
 
+func mergeRelations(currentRelations map[string]string, newRelations map[string]string) map[string]string {
+	// For keys in newRelations:
+	// if key not in currentDoc, add key/value
+	// if key in currentDoc, if value == "", remove
+	// if key in currentDoc, if value != "", replace
+	for id, value := range newRelations {
```

Comment:
Can you please rename `value` to `newValue`; it makes the logic below easier to follow

Reply:
done